contents/blog/startup-finance-without-finance.md

```diff
@@ -0,0 +1,210 @@
+- Google Sheets
+
+## Contents
+
+- Keeping score 
+- Predicting the future
+- Delaying the inevitable
+
+> Who to bank with is not covered in this guide, but please pick something startup-appropriate like Brex or Mercury in the US, or Revolut in Europe.
+
+## Keeping score - aka Accounting
+
+Ok, let's start with the good stuff. You need to hire an accountant. Specifically, you need [Fondo](https://www.tryfondo.com/). They will handle:
+
+- _Bookkeeping_ - this means they record all money that goes in and out of your company in QuickBooks and assign it to the right place, such as marketing, hosting etc.
+- _Financial reports_ - monthly, quarterly, and annual financial reports. More on these in a minute. 
+- _Tax filings_ - there are a myriad of state and federal taxes to pay. You _will_ forget the deadlines. Don't do this yourself - the fines are brutal if you are late. 
+- _Tax credits_ - exciting bonus money the government gives back to you if you fill out enough forms. Pay Fondo a commission and they will do it for you - it's basically free money. 
+
+(At this point, you'll probably want to step out of this guide, contact Fondo, and then come back. I'll wait.)
+
+The next step here is to set your accountant up with the necessary access to your QuickBooks account and probably your bank account(s) and things like Stripe. They will tell you what they need. 
+
+**Monthly accounting process**
+
+Once you're all set up, this is how we run our accounting process each month:
+
+1. Around the 20th of the month, we receive a monthly financial report from Fondo with a bit of explanatory blurb. Typically this will include a few questions about where to categorize certain transactions - answer these asap. 
+2. We read the report really carefully. Is there anything that doesn't make sense? You should be able to explain every big change you see from the previous month. If you can't, log into QuickBooks and look at your [Sales](https://app.qbo.intuit.com/app/sales) or [Expenses](https://app.qbo.intuit.com/app/expenses) tab. 
+3. If there are any changes needed, ask Fondo to make them - we usually have at least a couple each month. 
+4. Compare how you did against your budget - we'll get to this later. 
+5. Share the report with your _entire_ team, and include a bit of commentary. This is a great habit to get into, as it gives transparency and encourages people to ask questions, which keep you honest!
+
+### How to read your financial report
+
+You'll sometimes hear these referred to as 'management accounts' by the way - for our purposes here, you can treat them as the same thing. When you get bigger and fancier, you can produce glossy, proper management accounts that include other nice graphs and metrics, but you don't need those yet. 
+
+Your report will comprise three parts - rather than listing out what every line item means, I'm just going to include particularly important or non-obvious things you should look out for:
```

Comment:
```suggestion
Your report (sometimes referred to as 'management accounts') will comprise three parts - rather than listing out what every line item means, I'm just going to include particularly important or non-obvious things you should look out for:
```